monitoring/noxfile.py

```diff
@@ -122,7 +122,8 @@ def system(session):
     session.install("-e", ".")
 
     # Additional setup for VPCSC system tests
-    if os.environ.get("GOOGLE_CLOUD_TESTS_IN_VPCSC") != "true":
+    in_vpc = os.environ.get("GOOGLE_CLOUD_TESTS_IN_VPCSC")
```

Comment:
We can avoid the special handling for `None` below by providing a default, e.g.:
```suggestion
    in_vpc = os.environ.get("GOOGLE_CLOUD_TESTS_IN_VPCSC", "false")
```